src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        StringBuilder builder = new StringBuilder();
+        builder.append("Figure: the radius is")
+                .append(radius)
+                .append(" the color is: ")
+                .append(color);
+        System.out.println(builder.toString());
+    }
```

Comment:
The class `Circle` should not extend `Figure` directly if `Figure` is intended to be an abstract class for setting behavior. If `Figure` is an interface, then it's correct to implement it. Make sure that the use of abstract classes and interfaces is clear and follows the principles of object-oriented programming.

Reply:
no